odl/space/npy_ntuples.py

```diff
@@ -74,12 +74,12 @@ def element(self, inp=None, data_ptr=None):
 
         Examples
         --------
-        >>> strings3 = NumpyNtuples(3, dtype='U1')  # 1-char strings
```

Comment:
Why remove this example?

Reply:
Because output is different for Python 2 and 3, too much hassle. The bool example does the same basically.